.circleci/config.yml

```diff
@@ -76,12 +76,18 @@ jobs:
           paths:
             - .
   checkout_tags:
-    <<: *build_config
+    <<: *deploy_config
     steps:
       - *restore_repo
-      - run: git config --global url."https://github.com/".insteadOf "git@github.com:"
-      - run: git config --global url."https://".insteadOf "git://"
-      - run: git fetch --tags
+      # Temporary workaround for https://discuss.circleci.com/t/22437
+      - run: |
+          if [ -n "$CIRCLE_TAG" ]
+          then
+            mkdir -p ~/.ssh
+            echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==' \
```

Comment:
Can you explain why it's okay to place this under version control?

Reply:
This is a line we probably all have in our ~/.ssh/known_hosts but I think it's better to verify the ssh fingerprint independently if possible before adding it to known_hosts

https://stackoverflow.com/a/54090063/48641